app/services/sufia/workflow/workflow_by_admin_set_strategy.rb

```diff
@@ -0,0 +1,16 @@
+module Sufia
+  module Workflow
+    class WorkflowByAdminSetStrategy
+      attr_reader :admin_set_id
```

Comment:
Not sure you need this here atm, though I'm generally :+1: on hiding ivars behind attr_readers.